js/src/components/google-combo-account-card/connected-ads-account-actions.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppNotice from '.~/components/app-notice';
+import ClaimAdsAccount from './claim-ads-account';
+
+/**
+ * Renders the actionable components for connected accounts.
+ * @param {Object} props Component props.
+ * @param {boolean} props.claimGoogleAdsAccount Whether the user should claim the Google Ads account.
+ * @param {boolean} props.showConversionMeasurementNotice Whether to show the conversion measurement notice.
+ * @return {JSX.Element} Connected accounts actions.
+ */
+const ConnectedAdsAccountsActions = ( {
+	claimGoogleAdsAccount,
+	showConversionMeasurementNotice,
+} ) => {
+	if ( ! claimGoogleAdsAccount && ! showConversionMeasurementNotice ) {
+		return null;
+	}
+
+	return (
+		<div className="gla-connected-ads-account-actions">
+			{ claimGoogleAdsAccount && <ClaimAdsAccount /> }
+
+			{ showConversionMeasurementNotice && (
+				<AppNotice
```

Comment:
Using the `Notice` imported from `'@wordpress/components'` should be enough as all `AppNotice` styles are overridden.

![image](https://github.com/user-attachments/assets/2d38f077-16a6-4579-a630-7e297be96583)

Reply:
👍